certified-connectors/Africa's Talking Voice/apiDefinition.swagger.json

```diff
@@ -0,0 +1,211 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Africa's Talking Voice",
+    "description": "Build dynamic scalable voice applications that reside entirely in the cloud.",
+    "version": "1.0",
+    "contact": {
+      "name": "Advice and answers from the Africa's Talking Team",
+      "url": "https://help.africastalking.com/en/",
+      "email": "voice@africastalking.com"
+    }
+  },
+  "host": "voice.africastalking.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/call": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "201",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "entries": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "phoneNumber": {
+                        "type": "string",
+                        "description": "phoneNumber"
+                      },
+                      "status": {
+                        "type": "string",
+                        "description": "status"
+                      },
+                      "sessionId": {
+                        "type": "string",
+                        "description": "sessionId"
+                      }
+                    }
+                  },
+                  "description": "entries"
+                },
+                "errorMessage": {
+                  "type": "string",
+                  "description": "errorMessage"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Call",
```

Comment:
@crispinkoech Please update it to "Make an outbound call" instead of just "Call".

Refer - [Documentatio](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)